README.md

```diff
@@ -1,40 +1,89 @@
 # md-to-pdf
 
-A web service for converting markdown to PDF
+**A modern web service for converting Markdown to PDF with customizable templates and improved fonts.**  
```

Comment:
```suggestion
**A web service for converting Markdown to PDF with customizable templates and improved fonts.**  
```

At some point it might no longer be.